### PR TITLE
Pipe component should never creates own token.

### LINKED ISSUE #188: There are multiple token creations in client
Should be reduced to one

### PR DESCRIPTION
**Pull request by iwanbk**
The client will be supplied by main client.

Fixes #188

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -38,22 +39,33 @@ func New(conf *config.Config) (*Client, error) {
 }
 
 func newClient(conf *config.Config) (*Client, error) {
+	nsMgr := itsyouonline.NewClient(conf.Organization, conf.IYOAppID, conf.IYOSecret)
+	iyoToken, err := nsMgr.CreateJWT(conf.Namespace, itsyouonline.Permission{
+		Write: true,
```
Comment (zaibon) [inline client/client.go]: We should ask for all permission, IYO will only return the scope that the user actually has anyway.

Comment (iwanbk) [inline client/client.go]: Why all? we only have read & write pipe

**Comment by zaibon** [comment]
https://github.com/zero-os/0-stor/issues/188 make sense, but what about the discussion regarding allowing unit test without the need of a jwt. With this PR, I don't think we make a move in that direction.

**Comment by iwanbk** [comment]
> make sense, but what about the discussion regarding allowing unit test without the need of a jwt. With this PR, I don't think we make a move in that direction.

This PR also wont harm the path to that direction

**Comment by iwanbk** [comment]
> This PR also wont harm the path to that direction

We actually move to that direction with this path, because the client only creates the token once. It means we only have one place to modify in case of we allow empty token

**Comment by zaibon** [comment]
Make sense.

Please rebase from master, seems the test fails since the change on the travis file on master

**Comment by iwanbk** [comment]
> Please rebase from master, seems the test fails since the change on the travis file on master

done

File: client/client.go (hunk 1 above)
Comment (zaibon) [inline client/client.go]: Inded, and that make me thing that we miss some method on the client then. 
Cause we also should be able to delete objects. But let's do that in another PR
